packages/material-ui-system/src/css.js

```diff
@@ -13,15 +13,32 @@ function omit(input, fields) {
   return output;
 }
 
+function handleCss(styleFunction, theme) {
```

Comment:
I'd prefer to inline this. `handleCss` does not improve readability since it's too generic i.e. what's being handled?

Reply:
Inlined the function